dashboards-observability/public/components/visualizations/charts/bar/bar_type.ts

```diff
@@ -28,8 +28,8 @@ export const createBarTypeDefinition = (params: any) => ({
   name: 'bar',
   type: 'bar',
   id: 'bar',
-  label: 'Bar',
-  fullLabel: 'Bar',
+  label: 'Vertical Bar',
```

Comment:
In all cases where two words are present, you are making the second word start with lower case. I agree about consistency. Then for `Vertical Bar` why the `B` is upper case in all places?

Reply:
Changed 'Vertical Bar' to 'Vertical bar'